java/pom.xml

```diff
@@ -344,9 +344,9 @@
           <execution>
             <id>analyze</id>
             <phase>verify</phase>
-            <goals>
+            <!--<goals>
               <goal>analyze-only</goal>
-            </goals>
+            </goals>-->
```

Comment:
why is this goal commented out ? Does it fail something ?

Reply:
@scampi yes the changes were made in this [commit](8fe1264ed37e74ce2be17d0daf77898db68bbfa0) and it occurs as `analyze-only` does stricter dependency analysis. It complains due to the following 
```
[WARNING] Unused declared dependencies found:
[WARNING]    org.apache.arrow:arrow-memory-netty:jar:15.0.0:test
[WARNING]    org.apache.arrow:arrow-memory-unsafe:jar:siren-15.0.0-2-SNAPSHOT:compile
[WARNING]    org.slf4j:jul-to-slf4j:jar:2.0.9:test
[WARNING]    org.slf4j:jcl-over-slf4j:jar:2.0.9:test
[WARNING]    org.slf4j:log4j-over-slf4j:jar:2.0.9:test
[WARNING]    org.junit.jupiter:junit-jupiter-engine:jar:5.10.1:test
[WARNING]    org.junit.vintage:junit-vintage-engine:jar:5.10.1:test
[WARNING]    org.junit.jupiter:junit-jupiter-params:jar:5.10.1:test
[WARNING]    org.mockito:mockito-junit-jupiter:jar:2.25.1:test
[WARNING]    ch.qos.logback:logback-classic:jar:1.3.14:test
[WARNING]    de.huxhorn.lilith:de.huxhorn.lilith.logback.appender.multiplex-classic:jar:0.9.44:test
 
```
An alternative to get rid of this error is by changing `<failOnWarning>true</failOnWarning>` to `false`

Reply:
hmm it feels less intrusive to change the value of the `failOnWarning` parameter

Reply:
Yes, I have uncommented the `analyze-only` goal and turned `failOnWarning`  to false - 505495eb5a06dcb3250e821c52ac033ba8e5498d and I have re-ran the other [pr](https://github.com/sirensolutions/siren-platform/pull/7058).